src/templates/header.jinja2

```diff
@@ -47,9 +47,10 @@ public:
     {{ fn_decl(fn, fn.name) }} override;
 {% endfor -%}
 
-{% if virtual_fns %}private:{% endif %}
+{% if virtual_fns %}{{ '\n' }}private:{{ '\n' }}{% endif %}
 {% for fn in virtual_fns %}
-    {{ fn_decl(fn, fn.name + '_Reversed') }};
+    {{ fn_decl(fn, fn.name + '_Reversed') }};{{ '\n' }}
 {%- endfor %}
 };
```

Comment:
```jinja
{% for fn in virtual_fns %}
    {{ fn_decl(fn, fn.name + '_Reversed') }};
{% endfor %}
```
`%-` - Removes whitespace/newlines from until the last character / `}}` `%}`
`-%` - Does something similar, it removes up until the first `{{` or {%`
So just remove it here, and it should work